app/src/main/java/org/thoughtcrime/securesms/messagerequests/MessageRequestsActivity.kt

```diff
@@ -0,0 +1,100 @@
+package org.thoughtcrime.securesms.messagerequests
+
+import android.app.AlertDialog
+import android.content.Intent
+import android.database.Cursor
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.core.view.isVisible
+import androidx.loader.app.LoaderManager
+import androidx.loader.content.Loader
+import dagger.hilt.android.AndroidEntryPoint
+import network.loki.messenger.R
+import network.loki.messenger.databinding.ActivityMessageRequestsBinding
+import org.thoughtcrime.securesms.BaseActionBarActivity
+import org.thoughtcrime.securesms.conversation.v2.ConversationActivityV2
+import org.thoughtcrime.securesms.database.ThreadDatabase
+import org.thoughtcrime.securesms.database.model.ThreadRecord
+import org.thoughtcrime.securesms.mms.GlideApp
+import org.thoughtcrime.securesms.mms.GlideRequests
+import org.thoughtcrime.securesms.util.push
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class MessageRequestsActivity : BaseActionBarActivity(), ConversationClickListener, LoaderManager.LoaderCallbacks<Cursor> {
```

Comment:
We might want to inherit from PasswordRequiredActionBarActivity here, or whatever it is called :thinking: although I'm not sure